ci/resource_types.yml

```
@@ -0,0 +1,5 @@
+resource_types:
+- name: pull-request
+  type: docker-image
+  source:
+    repository: jtarchie/pr
```

We are now using `teliaoss/github-pr-resource` as `jtarchie/pr` is deprecated.  This means some things will have changed when calling this resource. All docker image resource types now require a version as standard.  See: https://github.com/telia-oss/github-pr-resource for reference

```suggestion
    repository: teliaoss/github-pr-resource
    tag: latest
```